sync/class.jetpack-sync-defaults.php

```diff
@@ -111,6 +111,7 @@ class Jetpack_Sync_Defaults {
 		'advanced_seo_front_page_description', // Jetpack_SEO_Utils::FRONT_PAGE_META_OPTION
 		'advanced_seo_title_formats', // Jetpack_SEO_Titles::TITLE_FORMATS_OPTION
 		'jetpack_api_cache_enabled',
+		'jetpack_api_base_domain', // usally not set to anything // unless Jetpack_Options::get_option( 'debug' ) is set to true.
```

Comment:
Why do we need/want to sync this value?

Reply:
So that we can see it change (log it to the activity log, potentially). 
Also it make it easier to change the value via the api.